pyocd/probe/pydapaccess/cmsis_dap_core.py

```diff
@@ -528,6 +534,6 @@ def vendor(self, index, data):
 
         if resp[0] != Command.DAP_VENDOR0 + index:
             # Response is to a different command
-            raise DAPAccessIntf.DeviceError()
+            raise DAPAccessIntf.DeviceError("expected DAP_VENDOR0")
```

Comment:
The error message should be `"expected DAP_VENDOR%i" % index`.

Reply:
Agreed; good catch.